addons/website/views/snippets/snippets.xml

```diff
@@ -782,5 +782,10 @@
             <we-button data-no-preview="true" data-open-custom-code-dialog="footer">&amp;lt;/body&amp;gt;</we-button>
         </we-row>
     </div>
+
+    <!-- Mobile display options -->
+    <div data-selector="section .row > div">
+        <we-checkbox string="invisible on mobile" data-select-class="d-none d-md-block" data-no-preview="true"/>
```

Comment:
- We try to have all labels to be "positive", so I would inverse with "Shown in mobile"
    - A we-checkbox toggles between two values for its related method, the two being separated with a `|` character: `data-select-class="A|B"` toggles between class A and B. (By default we considerer `A=''`). So to have a "Shown in mobile" version, you can declare this as `data-select-class="d-none d-md-block|"`
- Use an uppercase starting letter